portal/server/app/route.ts

```diff
@@ -38,24 +36,18 @@ export async function GET(req: Request) {
 
     const atBaseUrl = portalDomain == url.host.split(':')[0]
     if (atBaseUrl) {
-        console.log('serving the landing page from another service')
-        const landingPageServiceName = `https://walrus.site`
-        const response = await fetch(`${landingPageServiceName}${parsedUrl?.path}`)
-        const data = await response.text()
-        console.log(response.headers.get('content-type'))
-
-        // Proxy requests for CSS and fonts
-        const proxiedData = data.replace(
-            /(href|src)="\/([^"]+)"/g,
-            `$1="${landingPageServiceName}/$2"`
-        )
-
-        // Return the fetched data as a response
-        return new Response(proxiedData, {
-            headers: {
-                'content-type': response.headers.get('content-type') ?? 'text/html'
+        console.log('Serving the landing page from walrus...')
+        const blobId = '55onty23j6xl6axb7z2o03t5zs6gmosw30qjb4lqr3t60ukc0a'
+        const resourcePath = parsedUrl?.path == '/index.html' ?
+            '/index-sw-enabled.html' :
+            parsedUrl?.path ?? '/index-sw-enabled.html'
```

Comment:
True! But I preferred to keep using the shared static files among the server side portals. 
Another option would be to refactor the code so that both the service worker and the server use as landing page the `index.html`, and the `index.html` of the sw will be renamed to `index-pre-sw-not-enabledhtml`

Reply:
I would prefer making a separate PR for such change, including only the refactoring.

Reply:
Created https://github.com/MystenLabs/walrus-sites/issues/179 just for this.

Reply:
I meant more that in the script, instead of `rm temp-landing-page/index.html`, we could do  `mv temp-landing-page/index-sw-enabled.html temp-landing-page/index.html`. then the published site loads correcly, and the static files are all the same.

Reply:
but we can follow up with #179